tests/test_watson.py

```diff
@@ -361,6 +376,28 @@ def test_start_two_projects(watson):
     assert watson.is_started is True
 
 
+def test_start_default_tags(watson):
+    content = u"""
+[default_tags]
+my project = A B
+    """
+
+    with mock.patch.object(ConfigParser, 'read', mock_read(content)):
+        watson.start('my project')
+        assert watson.current['tags'] == ['A', 'B']
+
+
+def test_start_default_tags_with_tags(watson):
```

Comment:
Isn't that a bit long? I worry it would get truncated in the test output, making it less informative at the end.